avocado/utils/pci.py

```diff
@@ -320,6 +320,164 @@ def get_driver(pci_address):
                 return line.rsplit(None, 1)[-1]
 
 
+def unbind(driver, full_pci_address):
+    """
+    Unbind the pci device(full_pci_address) from driver
+
+    :param driver: driver of the PCI address (full_pci_address)
+    :param full_pci_address:  Full PCI address including domain (0000:03:00.0)
+    return: None
+    """
+    genio.write_file(f"/sys/bus/pci/drivers/{driver}/unbind", full_pci_address)
```

Comment:
Writing to this file might lead to raising the `OSError`, which could confuse the user of this util. IMO, it would be better to add `Not able to unbind {full_pci_address} from {driver}` message to those errors to increase future debugging.

Reply:
I can see the same problem to other utilities in this PR. Please update them as well.